modules/videojsVideoProvider.js

```diff
@@ -146,8 +146,9 @@ export function VideojsProvider(providerConfig, vjs_, adState_, timeState_, call
     // ~ Sort of resolved check if the player has a source to tell if the placement is instream
     // Still cannot reliably check what type of placement the player is if its outstream
     // i.e. we can't tell if its interstitial, in article, etc.
+    // update: cannot infer instream ever, always need declarations
     if (player.src()) {
-      video.placement = PLACEMENT.INSTREAM;
```

Comment:
is the idea to entirely remove the use of `placement` since it is deprecated ?  Or to include `plcmt` as well ?

Reply:
I think the idea is we want to provide information we can inder and not cross a line into invalid request territory on either field guessing too much

Reply:
@fowler446 had flagged this adapter in https://github.com/prebid/Prebid.js/issues/10452#issuecomment-2093546373; this pr is hoping to solve it for one of the last remaining adapters on the list